src/App.vue

```diff
@@ -218,8 +227,27 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
 
   private setResponsiveClassDebounced = debounce(this.setResponsiveClass, 250);
 
+  handleLocalStorageExceedWarning() {
+    this.showErrorLocalStorageExceed = true;
+  }
+
+  clearLocalStorage() {
+    // TODO delete later
+    // localStorage.removeItem('fillerKey');
+
+    const keysToRemove: string[] = [];
+    for (let i = 0; i < localStorage.length; i++) {
+      const key = localStorage.key(i);
+      if (key && (key.includes('.assetsAddresses') || key.includes('.history'))) {
```

Comment:
The possible list to remove:
`.assetsAddresses`
`.history`
`.ethBridgeHistory`
`.evmHistory`
`.subHistory`

`list.some((item) => key.includes(item))`